openweathermap.go

```diff
@@ -68,6 +71,17 @@ var LangCodes = map[string]string{
 	"ZH_CN": "Chinese Simplified",
 }
 
+// Exclude holds all supported excludes option to be used
+type Exclude string
+
+const (
+	ExcludeCurrent  Exclude = "current"
```

Comment:
1. Is it necessary for these to be typed? Can they be native strings? If not, let's not add the additional type layer.
2. Do these need to be exported for use outside of this library? If not, let's unexport them.

Reply:
The type isn't used so we should keep it simple and make it a string until a specific type is needed.